### PR TITLE
uploading profile picture

### PR DESCRIPTION
**Pull request by Thomas905**

### CODE REVIEW

File: assets/styles/app.scss [hunk 1]
```diff
@@ -9,4 +9,3 @@
 @import 'profile';
 @import 'student';
 @import 'mentor';
```
Comment (MorganeL-pro) [inline assets/styles/app.scss]: Maybe you should do a git pull before a push to avoid conflicts.

**Review by MorganeL-pro — COMMENTED** [review]
You should do a git pull before a push to avoid conflicts (css files...)
Pictures of users should not be a the repo. You shoud add them to gitignore.

File: composer.json [hunk 2]
```diff
@@ -44,6 +44,7 @@
         "symfony/web-link": "5.3.*",
         "symfony/webpack-encore-bundle": "^1.11",
         "symfony/yaml": "5.3.*",
+        "symfonycasts/verify-email-bundle": "^1.7",
         "twig/extra-bundle": "^2.12|^3.0",
         "twig/twig": "^2.12|^3.0"
     },
```
Comment (elodiedaubie) [inline composer.json:50]: Hello, I don't understand why we have this file here, It should not be in this PR

Comment (elodiedaubie) [reply composer.json:50]: same for the composer.lock

File: config/bundles.php [hunk 3]
```diff
@@ -14,4 +14,5 @@
     Symfony\Bundle\MakerBundle\MakerBundle::class => ['dev' => true],
     Symfony\WebpackEncoreBundle\WebpackEncoreBundle::class => ['all' => true],
     Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle::class => ['dev' => true, 'test' => true],
+    SymfonyCasts\Bundle\VerifyEmail\SymfonyCastsVerifyEmailBundle::class => ['all' => true],
```
Comment (elodiedaubie) [inline config/bundles.php:17]: This file should not be there as well

File: migrations/Version20220104134712.php [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE user ADD is_verified TINYINT(1) NOT NULL');
```
Comment (elodiedaubie) [inline migrations/Version20220104134712.php:23]: this file should not be here

**Review by elodiedaubie — CHANGES_REQUESTED** [review]
Hello Thomas, I think that you PR needs some cleaning because there are 36 files in it. Some of the files (migrations files for ex) have nothing to do with it

**Pull request closed without merging**